datasync/chaindatafetcher/kas/model.go

```diff
@@ -1,6 +1,9 @@
 package kas
 
-const TxTableName = "klay_transfers"
+const (
+	TxTableName         = "klay_transfers"
```

Comment:
It is weird that the value of "Tx"TableName is klay_"transfers".
Let's use "transactions" for Klaytn transactions.

```suggestion
	TxTableName         = "klay_transactions"
```

Reply:
Agreed. However, it is the actual table name and `gorm` insert the data into the table with the given name.